src/cli.ts

```
@@ -19,6 +19,7 @@ cli
   .option('-e, --env <environments>', 'the conditional environment resolutions to apply')
   .option('-m, --map <map>', 'an authoritative initial import map', { default: 'importmap.json' })
   .option('-p, --provider <proider>', 'the default provider to use for a new install, defaults to `jspm`', { default: 'jspm' })
+  .option('--cache [mode]', 'the default provider to use for a new install, defaults to `jspm`', { default: true })
```

The use cases here are `cache: 'offline'` and `cache: false` to support. `--cache offline` as an explicit string seems fine, but we should warn on other strings that it won't do anything. `cache: false` also doesn't seem to be supported?

---

for cache false the user should do `--no-cache`.

---

We should update the description I think, then also say that `--cache=offline` is supported, and also maybe throw a nice error if passing `--cache=anystring` since this won't have defined behaviour.